composer.json

```diff
@@ -3,74 +3,79 @@
+        "symfony/polyfill-apcu": "^1.4",
+        "symfony/security-bundle": "^3.3",
+        "symfony/swiftmailer-bundle": "^3.0",
+        "symfony/templating": "^3.3",
+        "symfony/translation": "^3.3",
+        "symfony/validator": "^3.3",
+        "symfony/yaml": "^3.3",
+        "twig/extensions": "^1.5",
+        "twig/twig": "^2.4",
+        "white-october/pagerfanta-bundle": "^1.0"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle"            : "^3.0",
-        "friendsofphp/php-cs-fixer"            : "^2.0",
-        "sensio/generator-bundle"              : "^3.0",
-        "symfony/phpunit-bridge"               : "^3.0"
-    },
-    "scripts": {
-        "symfony-scripts": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
-        ],
-        "post-install-cmd": [
-            "@symfony-scripts"
-        ],
-        "post-update-cmd": [
-            "@symfony-scripts"
-        ]
+        "dama/doctrine-test-bundle": "^3.1",
+        "friendsofphp/php-cs-fixer": "^2.4",
+        "symfony/debug-bundle": "^3.3",
+        "symfony/dotenv": "^3.3",
+        "symfony/phpunit-bridge": "^3.3",
+        "symfony/profiler-pack": "^1.0",
+        "symfony/var-dumper": "^3.3",
+        "symfony/web-server-bundle": "^3.3"
     },
     "config": {
         "platform": {
-            "php": "5.5.9"
+            "php": "7.1.3"
+        },
+        "preferred-install": {
+            "*": "dist"
         },
         "sort-packages": true
     },
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\Tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "auto-scripts": {
+            "make cache-warmup": "script",
+            "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
+            "security-checker security:check": "script"
+        }
+    },
+    "conflict": {
+        "symfony/symfony": "*",
+        "symfony/twig-bundle": "<3.3",
+        "symfony/debug": "<3.3"
+    },
     "extra": {
-        "symfony-app-dir": "app",
-        "symfony-bin-dir": "bin",
-        "symfony-var-dir": "var",
-        "symfony-web-dir": "web",
-        "symfony-tests-dir": "tests",
-        "symfony-assets-install": "relative",
-        "incenteev-parameters": {
-            "file": "app/config/parameters.yml"
+        "symfony": {
+            "id": "01BP22MQ00QH5WD6NN6HHD3EX4",
+            "allow-contrib": false
```

Comment:
Maybe set allow-contrib to `true`?
So if people use the demo and try to add a package/bundle which is available in contrib recipes, it will work just fine?
Just an idea, I'm not sure.